Gopkg.lock

```diff
@@ -3,19 +3,17 @@
 
 [[projects]]
   branch = "master"
-  digest = "1:e4c72127d910a96daf869a44f3dd563b86dbe6931a172863a0e99c5ff04b59e4"
+  digest = "1:cabe9ec8289704e71e45138c18ad90275bce25a8d61145c4130b55d9d9a6d6dc"
   name = "github.com/Sirupsen/logrus"
   packages = ["."]
   pruneopts = "UT"
-  revision = "dae0fa8d5b0c810a8ab733fbd5510c7cae84eca4"
+  revision = "9b3cdde74fbe9443d704467498a7dcb61a79de9b"
 
 [[projects]]
-  branch = "master"
-  digest = "1:3d070a9f6bc767f4f4059c41a1d9d6b796dc4b1297913b4b9f6d38dc89b095b9"
+  digest = "1:deee6e2dfcb7987637f3dd4c0fae851e90f588957c3f46d7e0c229bcc8d8f6d6"
   name = "github.com/bitrise-core/bitrise-init"
```

Comment:
There are both references to "github.com/bitrise-core/bitrise-init" (line 14) and ""github.com/bitrise-io/bitrise-init" (line 47)

Reply:
unfortunately I will only be able to fix that if I merge these changes to master and then I regenerate the deps. This is because the current master on this repository uses references to the old bitrise-core one. (So it fetches dependency of dependencies as well.

Reply:
👍